src/mame/mess.flt

```diff
@@ -38,6 +38,7 @@ amstrad/pc1512.cpp
 amstrad/pcw.cpp
 amstrad/pcw16.cpp
 amstrad/pda600.cpp
+amstrad/pda600_copro.cpp
 apf/apf.cpp
```

Comment:
Only system driver sources should be in filter files.  If it can’t find the device sources automatically you have bigger problems.